### PR TITLE
feat: Make repair and refreshDimensions configurable in restoreElements

### PR DESCRIPTION
**Pull request by ad1992**
fixes https://github.com/excalidraw/excalidraw/issues/6237

Will update the DOCS in next branch.

### CODE REVIEW

**Comment by dwelle** [comment]
`repair` is too generic. Could mean repair anything, e.g. text dimensions (which it does not contain). So `repairBindings`.

File: src/packages/excalidraw/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,24 @@ The change should be grouped under one of the below section and must contain PR
 Please add the latest change on the top under the correct section.
 -->
 
+## Unreleased
+
+### Features
+
+- [`restoreElements`](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/api/utils/restore#restoreelements) API now takes an optional parameter `opts` which currentl supports the below attributes
```
Comment (dwelle) [inline src/packages/excalidraw/CHANGELOG.md:18]:
```suggestion
- [`restoreElements`](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/api/utils/restore#restoreelements) API now takes an optional parameter `opts` which currently supports the below attributes:
```

File: src/packages/excalidraw/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,24 @@ The change should be grouped under one of the below section and must contain PR
...
+{ refreshDimensions?: boolean, repair?: boolean }
+```
+
+The same `opts` param has been added to [`restore`](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/api/utils/restore#restore) API as well.
+
+For more details refer to the [docs](https://docs.excalidraw.com)
+
+#### BREAKING CHANGE
+
+- The optional parameter `refreshDimensions` in[`restoreElements`](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/api/utils/restore#restoreelements) has been removed and can be enabled via `opts`
```
Comment (dwelle) [inline src/packages/excalidraw/CHANGELOG.md:30]:
```suggestion
- The optional parameter `refreshDimensions` in [`restoreElements`](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/api/utils/restore#restoreelements) has been removed and can be enabled via `opts`
```

**Comment by ad1992** [comment]
@dwelle this should be ready to merge

**Review by dwelle — APPROVED** [review]

**Pull request merged**